src/graphql/models/Article.js

```diff
@@ -580,6 +581,25 @@ const Article = new GraphQLObjectType({
           },
         }),
     },
+    contributors: {
+      type: new GraphQLList(Contributor),
+      description: 'Transcript contributors of the article',
+    },
+    transcribedAt: {
+      type: GraphQLString,
+      description: 'Time when the article was last transcribed',
+      resolve: async ({ contributors }) => {
+        if (!contributors || contributors.length === 0) {
+          return null;
+        }
+        const maxUpdatedAt = contributors.reduce(
+          (max, contributor) =>
+            contributor.updatedAt > max ? contributor.updatedAt : max,
+          contributors[0].updatedAt
```

Comment:
nit: can use `contributors.map(({updatedAt}) => updatedAt).reduce()`
so that we don't need to specify the initial value -- it will use the first item in the array when initial value is not specified ([doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce#initialvalue)).

Reply:
I use Math.max instead in the end.